init

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
+set -euo pipefail
+shopt -s expand_aliases
+
+cd "$(dirname "${0}")"
```

Comment:
I have a feeling this will fail when running the script using `./init`. Will test and change if necessary.